### ISSUE TITLE
[Docs] Define process for op bringup (advices and core requirements)

### ISSUE THREAD

**Issue by nvukobratTT** [tenstorrent/tt-forge-fe#155]

**Comment by staylorTT** [comment]
@nvukobratTT Was this defined?

**Comment by nvukobratTT** [comment]
No, we don't have any section outlining how to add new op in our docs:
- https://docs.tenstorrent.com/tt-forge-fe/

**Comment by nvukobratTT** [comment]
Note: This should also be covered through op bringup docs:
- https://github.com/tenstorrent/tt-forge-fe/issues/83#issuecomment-2619125376

**Comment by skhatriTT** [comment]
@nvukobratTT can we close this?

**Comment by nvukobratTT** [comment]
> [@nvukobratTT](https://github.com/nvukobratTT) can we close this?

Yes.
